### ISSUE TITLE
Add stack support

### ISSUE THREAD

**Issue by NCrashed** [indicatrix/hs-geojson#11]
Hi! Thank you for awesome library!

The package isn't buildable with [stack](www.stackage.org) either current stable image `lts-3.2`, either latest nightly `nightly-2015-08-27`.

stack.yml:

```
flags: {}
packages:
- '.'
extra-deps:
  - geojson-1.3.0
  - validation-0.5.1
resolver: nightly-2015-08-27
```

```
--  Failure when adding dependencies:    
      aeson: needed (==0.9.*), latest is 0.9.0.1, but 0.8.0.2 found
      lens: needed (==4.11.*), latest is 4.12.3, but 4.12.3 found
    needed for package: geojson-1.3.0

--  Failure when adding dependencies:    
      geojson: needed (-any), latest is 1.3.0, but couldn't resolve its dependencies
    needed for package: new-template-0.1.0.0
```

Are there any plans to publish the library to stackage or relax version bounds to fit LTS?

**Comment by domdere** [comment]
There were no explicit plans to publish the library to stackage since i dont really use it, 

Not averse to the idea putting it up there though.

I would be pretty reliant on ppl like yourself to speak up when compatibility is broken though cos I'll probably never notice.

Thanks for the PR, dont really have a problem with relaxing the version bounds, as long as upper bounds do exist.

**Comment by NCrashed** [comment]
> Not averse to the idea putting it up there though.

The major problem for publishing at stack is `validation` dependency. As soon as the library is at stackage the geojson also could be published. I'll check out validation for the case.

> I would be pretty reliant on ppl like yourself to speak up when compatibility is broken though cos I'll probably never notice.

Never mind, I always whine to an issue tracker when face build problems.

**Comment by newmana** [comment]
The fix for the validation was fixed in 1.3.3.

http://hackage.haskell.org/package/geojson

And now part of the nightly builds:
https://www.stackage.org/nightly-2018-09-20/package/geojson-1.3.3
